app/scripts/migrations/135.ts

```diff
@@ -0,0 +1,101 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+import type { SmartTransaction } from '@metamask/smart-transactions-controller/dist/types';
+
+export type VersionedData = {
+  meta: {
+    version: number;
+  };
+  data: {
+    PreferencesController?: {
+      preferences?: {
+        smartTransactionsOptInStatus?: boolean | null;
```

Comment:
Why is the `smartTransactionsOptInStatus` line here twice?

Reply:
Thanks for the heads up, I will most likely be removing the root level property in the `controllerMetadata` inside `preferences-controller` and that should fix this. I was initially trying to figure out the right place for it and this is a valid point.